google/cloud/pubsub/schema.h

```diff
@@ -41,12 +41,12 @@ class Schema {
       : project_id_(std::move(project_id)), schema_id_(std::move(schema_id)) {}
 
   /// @name Copy and move
```

Comment:
Q. Is there any difference between the `@name` being inside or outside the group?

Reply:
I believe not.  I think we need to pick one style to simplify global changes, probably "inside the group", but not in this series of PRs.